allow the use of Reliable Stream Resets (#58)
kazuho/draft-kazuho-quic-quic-services-for-streams: draft-ietf-quic-qmux.md

if the transport is blocked by flow or congestion control.

> ## Reliable Stream Reset Extension

Reviewer: ```suggestion
## Stream Resets with Partial Delivery Extension
```

> The use of the Reliable Stream Reset Extension

Reviewer: ```suggestion
The use of the Stream Resets with Partial Delivery Extension
```

> {{!RELIABLE_RESET=I-D.ietf-quic-reliable-stream-reset}} is permitted. The

Reviewer: ```suggestion
{{!RESET-STREAM-AT=I-D.ietf-quic-reliable-stream-reset}} is permitted. The
```
